features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -978,6 +978,53 @@ CREATE TABLE IF NOT EXISTS IDN_FED_AUTH_SESSION_MAPPING (
 	PRIMARY KEY(IDP_SESSION_ID)
 );
 
+CREATE TABLE IF NOT EXISTS IDN_CONFIG_TYPE (
+ID          VARCHAR(255)  NOT NULL,
```

Comment:
it seems there is a formatting issue when we compare with other tables in this file